src/controller/CommissioningDelegate.h

```diff
@@ -527,6 +558,9 @@ class CommissioningParameters
         mDAC.ClearValue();
         mTimeZone.ClearValue();
         mDSTOffsets.ClearValue();
+        mICDCheckInNodeId.ClearValue();
```

Comment:
mICDCheckInNodeId is not external-buffer-dependent, is it?  Neither are mICDMonitoredSubject or mICDCounter.

mICDSymmetricKey is, perhaps, if it's actually used as input (which would be odd), but then we're not handling it right in AutoCommissioner and will get use-after-free.

What's the actual expected lifetime setup here for the mICDSymmetricKey?  Why is it stored in commissioning parameters anyway, and for how long?